code/modules/events/holiday/easter.dm

```diff
@@ -66,7 +66,7 @@
 
 //Bunny Suit
 /obj/item/clothing/head/costume/bunnyhead
-	name = "Easter Bunny Head"
+	name = "Easter Bunny head"
```

Comment:
Should have `\improper`?

Reply:
If this is capitalized, why are the other parts of the costume not capitalized?